### DISCUSSION TITLE
Reorder + Rename text effects?

### DISCUSSION THREAD

**Discussion by hellocatfood** [friction2d/friction#131] [Q&A] [answered]
Is there a way to reorder and rename text effects? I'm currently doing a text animation project and have a few text effects, some affecting letters and some affecting words. The effects all appear as `text effect` and I can't seem to rename or reorder them. The reordering matters I think because each effect is applied from top to bottom

**Comment by rodlie** [comment, marked as answer]
Hi,

Effect labels and order are fixed sadly. Thanks for reminding me, as this is of course something that should be fixed (and annoyed me in the past, but forgot).

Note that this might be a rabbit hole, as all effects behave the same .... :)

**Comment by rodlie** [comment]
Created an issue based on this.

https://github.com/friction2d/friction/issues/132

**Comment by hellocatfood** [comment]
Thanks! And thanks for forking/maintaining this software
